### ISSUE TITLE
Pause physics without pausing animations

### ISSUE THREAD

**Issue by gintsmurans** [cocos2d/cocos2d-objc#657]
It would be great to have an option to pause only physics so the animations still work. My node structure looks like this:
CCScene
    -- CCPhysicsNode
        -- CCSpriteBatchNode
            -- Game elements

When a user hits pause button, CCPhysicsNode is paused and transparent pause layer slides down. It also pauses every child node CCPhysicsNode have recursively.  It looks way better if animations like rotating coins are still rotating. To achieve that I am currently extending CCPhysicsNode. Do you think cocos2d needs this feature built-in?

**Comment by jjxtra** [comment]
I second this. Perhaps allowing to pause nodes and their children rather than the whole scene?

**Comment by slembcke** [comment]
@jjxtra You are describing CCNode.paused. New as of v3 and the rewritten CCScheduler. A node is considered paused if it, or any of it's ancestors are paused.

@gintsmurans Do you mean you want something separate from the CCNode.paused property where you can pause the physics but not the child node's timers/actions? That would be mostly easy. Nodes would still receive update: and fixedUpdate: events even though the physics would not be running. I'm not sure if that's a problem or not... I'll have to think about it.

**Comment by gintsmurans** [comment]
Yup, thats exactly what I am thinking of. Its easy to implement, but it was confusing not to have this option built-in. Here is my gist, if anyone else is looking how to do it: https://gist.github.com/gintsmurans/9702934
